avantis_trader_sdk/rpc/asset_parameters.py

```diff
@@ -39,7 +39,8 @@ async def get_oi_limits(self):
         pairs_info = await self.client.pairs_cache.get_pairs_info()
         calls = []
         for pair_index in range(len(pairs_info)):
-            call_data = PairStorage.encodeABI(fn_name="pairMaxOI", args=[pair_index])
+            tx = PairStorage.functions.pairMaxOI(pair_index).build_transaction()
+            call_data = tx['data']
```

Comment:
This gives an error

````
\rpc\asset_parameters.py", line 43, in get_oi_limits
    call_data = await tx['data']
                      ~~^^^^^^^^
TypeError: 'coroutine' object is not subscriptable
sys:1: RuntimeWarning: coroutine 'AsyncContractFunction.build_transaction' was never awaited
```

Might need to add an await statement

Reply:
All errors fixed,,